### PR TITLE
hm6-fin

### PR DESCRIPTION
**Pull request by Levkosvd**

### CODE REVIEW

File: src/main/java/core/basesyntax/Excavator.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public  void doWork() {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Excavator.java]:
```suggestion
    public void doWork() {
```

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public  void doWork() {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Bulldozer.java]:
```suggestion
    public void doWork() {
```

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println("The bulldozer is started. Br br br br !");
+    }
+
+    public  void stopWork() {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Bulldozer.java]:
```suggestion
    public void stopWork() {
```

File: src/main/java/core/basesyntax/Excavator.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println("The excavator is started. Brom bom bom !");
+    }
+
+    public  void stopWork() {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Excavator.java]:
```suggestion
    public void stopWork() {
```

File: src/main/java/core/basesyntax/Excavator.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println("The exacavator stoped");
+    }
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Excavator.java]: remove redundant line

File: src/main/java/core/basesyntax/MainApp.java [hunk 6]
```diff
@@ -12,5 +12,18 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck1 = new Track();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        Machine truck = new Track();
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 7]
```diff
@@ -12,5 +12,18 @@
...
+        Machine bulldozer2 = new Bulldozer();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        Machine bulldozer = new Bulldozer();
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 8]
```diff
@@ -12,5 +12,18 @@
...
+        Machine exacavator3 = new Excavator();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        Machine excavator = new Excavator();
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 9]
```diff
@@ -12,5 +12,18 @@
...
+        Machine[] vechicle = new Machine[]{ truck1, bulldozer2, exacavator3};
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        Machine[] vehicles = new Machine[]{ truck1, bulldozer2, exacavator3};
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 10]
```diff
@@ -12,5 +12,18 @@
...
+        for (Machine v:vechicle) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]:
```suggestion
        for (Machine machine : vehicles) {
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 11]
```diff
@@ -12,5 +12,18 @@
...
+            v.doWork();
+        }
+        System.out.println("Three hours later...");
+        for (Machine v:vechicle) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]: the same

File: src/main/java/core/basesyntax/MainApp.java [hunk 12]
```diff
@@ -12,5 +12,18 @@
...
+            v.stopWork();
+        }
+    }
 }
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java]: is there only one blank line at the end of file?

Comment (Levkosvd) [reply src/main/java/core/basesyntax/MainApp.java]: why didn't maven see this?

Comment (Sofasmile) [reply src/main/java/core/basesyntax/MainApp.java]: I don't know) 
What about other comments?

**Pull request closed without merging**